packages/rpc-core/src/rpc-methods/common.ts

```diff
@@ -40,3 +45,25 @@ export type Base64EncodedDataResponse = [Base64EncodedBytes, 'base64'];
 export type Base64EncodedZStdCompressedDataResponse = [Base64EncodedZStdCompressedBytes, 'base64+zstd'];
 
 export type Base58EncodedTransactionSignature = string & { readonly __base58EncodedSignature: unique symbol };
+
+type TokenAmount = Readonly<{
+    amount: StringifiedBigInt;
+    decimals: number;
+    uiAmount?: number;
```

Comment:
This is actually `number | null`, which could cause problems if someone expects it to be a valid `number` whenever it's defined, rather than defined but `null`

https://docs.solana.com/api/http#token-balances-structure

Reply:
Thankyou, fixed!